src/main/java/frc/robot/Robot.java

```diff
@@ -17,17 +17,15 @@
 public class Robot extends TimedRobot {
   private Command autonomousCommand;
 
-  private RobotContainer robotContainer;
+  private RobotContainer robotContainer = new RobotContainer();
```

Comment:
Is there a reason to do this before ``robotInit``? Supposedly the ``robotInit`` is the right function for initialization code?

Reply:
I don't remember why I did this. I think you're right, let's change it back to that